pkg/importer/http-datasource.go

```diff
@@ -133,7 +133,7 @@ func (hs *HTTPDataSource) Info() (ProcessingPhase, error) {
 	if hs.contentType == cdiv1.DataVolumeArchive {
 		return ProcessingPhaseTransferDataDir, nil
 	}
-	if !hs.readers.Convert {
+	if hs.readers.Archived {
 		return ProcessingPhaseTransferDataFile, nil
```

Comment:
I say we transfer to scratch for archived as well so I think we can delete this `if`

Reply:
I fear we might uncover issues like the one this PR addressed (https://github.com/kubevirt/containerized-data-importer/pull/2845), but I'm ok with the change. Let's see how tests behave.